source/Cosmos.System2_Plugs/System/IO/DirectoryInfoImpl.cs

```diff
@@ -16,13 +13,13 @@ namespace Cosmos.System_Plugs.System.IO
     public static class DirectoryInfoImpl
     {
         /* The real implementation uses IEnumerable and do a conversion ToArray that crashes IL2CPU */
-        public static DirectoryInfo[] GetDirectories(DirectoryInfo aThis)
+        public static DirectoryInfo[] GetDirectories(DirectoryInfo thisContext)
```

Comment:
I don't like this naming scheme, can we just call it context or directory?

Reply:
K